modules/distr-tracing-config-otel-collector.adoc

```diff
@@ -29,6 +29,10 @@ metadata:
   namespace: tracing-system
 spec:
   mode: deployment
+  ports:
```

Comment:
This will not be needed if we implement [this](https://github.com/open-telemetry/opentelemetry-operator/issues/1689), right? Just to confirm

Reply:
correct